libpod/container.go

```diff
@@ -313,6 +313,9 @@ type ContainerConfig struct {
 	// ExitCommand is the container's exit command.
 	// This Command will be executed when the container exits
 	ExitCommand []string `json:"exitCommand,omitempty"`
+	// LocalVolumes are the built-in volumes we get from the --volumes-from flag
+	// It picks up the built-in volumes of the container used by --volumes-from
+	LocalVolumes []string
```

Comment:
We went through the trouble of parsing these in `cmd/podman/`.  Should we pass them through as an array of `struct`s here?  Then we don't have to duplicate parsing in `libpod`.

Reply:
We want to leave the parsing in libpod as well since libpod is an API and if the parsing is done in `cmd/podman` then varlink for example would fail to work as expected in this case. The parsing in podman is just to ensure that any bad input from the user is caught before we reach the libpod code.